zebra-state/src/service.rs

```diff
@@ -471,7 +471,7 @@ impl StateService {
         let queued_height = checkpoint_verified.height;
 
         // If we're close to the final checkpoint, make the block's UTXOs available for
-        // full verification of non-finalized blocks, even when it is in the channel.
+        // full verification of checkpoint blocks, even when it is in the channel.
```

Comment:
This is about taking the UTXOs of checkpoint verified blocks, and making them available for semantic verification:

```suggestion
        // semantic block verification, even when it is in the channel.
```